### PR TITLE
Issue #4208: Split and Organize Checkstyle inputs by Test for StaticVariableName

### PR DESCRIPTION
**Pull request by AkshitaKukreja30**
PR for Issue #4208 -
InputStaticVariableName.java was moved to staticvariablename

### CODE REVIEW

**Comment by rnveach** [comment]
@AkshitaKukreja30 CI is failing. Please fix and look over your work as there is some small naming mistakes.

**Comment by AkshitaKukreja30** [comment]
@rnveach Please review the PR

File: pom.xml [hunk 1]
```diff
@@ -506,7 +506,41 @@
               <propertyExpansion>project.basedir=${project.basedir}</propertyExpansion>
               <sourceDirectory>${project.basedir}</sourceDirectory>
               <includes>**/*</includes>
-              <excludes>src/main/**/*,src/test/java/**/*,src/site/resources/images/**/*,target/**/*,bin/**/*,.git/**/*,.idea/**/*,.settings/**/*,.checkstyle/**/*,.DS_Store/**/*,.nondex/**/*,.pmd/**/*,.fbExcludeFilterFile/**/*,.externalToolBuilders/**/*,.pmdruleset.xml,nbactions.xml,nb-configuration.xml,*.iml</excludes>
+              <!-- All .gitignore files should be added to exclusions -->
```
Comment (rnveach) [inline pom.xml]: We are not expecting any changes except in StaticVariableName's test and it's inputs.
Please remove all these other changes.

**Comment by AkshitaKukreja30** [comment]
@rnveach  Sorry about that. It happend because of rebasing to latest master. 
Can you please tell me how to rebase to latest master without having commits included in mine so that TeamCity build doesn't fail.

**Comment by AkshitaKukreja30** [comment]
@rnveach  Please review the changes.

**Comment by romani** [comment]
TC failures were unrelated, I restarted TC - passed

**Pull request merged**